s/features/notes/components/notes/xiome-notes.ts

```diff
@@ -1,160 +1,170 @@
-
-import styles from "./xiome-notes.css.js"
-import {makeNotesModel} from "../../models/notes-model.js"
-import {renderOp} from "../../../../framework/op-rendering/render-op.js"
-import {ComponentWithShare, mixinStyles, html} from "../../../../framework/component.js"
-import {ops} from "../../../../framework/ops.js"
+import styles from "./xiome-notes.css.js" 
+import {makeNotesModel}	from "../../models/notes-model.js" 
+import { renderOp }	from "../../../../framework/op-rendering/render-op.js" 
+import {ComponentWithShare,	mixinStyles, html} from "../../../../framework/component.js" 
+import { ops } from "../../../../framework/ops.js" 
 
 @mixinStyles(styles)
 export class XiomeNotes extends ComponentWithShare<{
-		notesModel: ReturnType<typeof makeNotesModel>
-	}> {
-
-	#model = this.share.notesModel
-	#cacheDetails = this.share.notesModel.createNotesCacheDetails()
-	#cache = this.#cacheDetails.cache
+	notesModel: ReturnType<typeof makeNotesModel>
+		}>{
+		#model = this.share.notesModel 
+		#cacheDetails = this.share.notesModel.createNotesCacheDetails() 
+		#cache = this.#cacheDetails.cache 
 
-	async init() {
+		async init() {
 		await this.#model.initialize()
 	}
 
 	subscribe() {
-		const unsubs = [
+	const unsubs = [
 			super.subscribe(),
 			this.#cacheDetails.setup(),
 			this.#cache.subscribe(() => this.requestUpdate()),
-		]
-		if (this.#model.isLoggedIn) {
-			this.#cache.fetchAppropriateNotes()
-		}
-		return () => unsubs.forEach(unsub => unsub())
+	] 
```

Comment:
there is a hidden spacebar after the `]` character, we want to avoid trailing whitespace